webserver/cWebem.cpp

```diff
@@ -2021,11 +2026,12 @@ namespace http {
 				// do normal handling
 				try
 				{
-					if (requestCopy.uri.find("/images/") == 0)
+					std::string uri = myWebem->ExtractRequestPath(requestCopy.uri);
+					if (uri.find("/images/") == 0)
 					{
-						std::string theme_images_path = myWebem->m_actTheme + requestCopy.uri;
+						std::string theme_images_path = myWebem->m_actTheme + uri;
 						if (file_exist((doc_root_ + theme_images_path).c_str()))
-							requestCopy.uri = theme_images_path;
+							requestCopy.uri = myWebem->GetWebRoot().c_str() + theme_images_path;
```

Comment:
As requestCopy.uri is a std::string, there is no need to call .c_str() (**myWebem->GetWebRoot().c_str()**)